### DISCUSSION TITLE
What is jedis release cycle

### DISCUSSION THREAD

**Discussion by zeekling** [xetorthio/jedis#3030] [Q&A] [answered]
1. How often will a major release be released?
2. How many maintenance releases?

@sazzad16

**Comment by sazzad16** [comment, marked as answer]
1. There is no decided _frequency_. Major release would be targeted whenever we have something major (and/or breaking) things to address.
2. For the latest major version, as long long we have commands and/or arguments left to cover and feasible things to support.

**Reply by zeekling** [reply]
> 1. There is no decided _frequency_. Major release would be targeted whenever we have something major (and/or breaking) things to address.
> 2. For the latest major version, as long long we have commands and/or arguments left to cover and feasible things to support.

ok.

**Comment by zeekling** [comment]
Will 3.5.x be maintained?
Or maintenance 3.9.x?
 @sazzad16

**Reply by sazzad16** [reply]
3.5.x - extremely unlikely to be maintained.

3.9.x - if there is a resolvable (severe) bug, yes.
